adagrams/game.py

```diff
@@ -1,11 +1,111 @@
+import random
+
+LETTER_TILES = {
+    'A': {'frequency': 9, 'value': 1},
+    'B': {'frequency': 2, 'value': 3},
+    'C': {'frequency': 2, 'value': 3}, 
+    'D': {'frequency': 4, 'value': 2}, 
+    'E': {'frequency': 12, 'value': 1}, 
+    'F': {'frequency': 2, 'value': 4}, 
+    'G': {'frequency': 3, 'value': 2}, 
+    'H': {'frequency': 2, 'value': 4}, 
+    'I': {'frequency': 9, 'value': 1}, 
+    'J': {'frequency': 1, 'value': 8}, 
+    'K': {'frequency': 1, 'value': 5}, 
+    'L': {'frequency': 4, 'value': 1}, 
+    'M': {'frequency': 2, 'value': 3}, 
+    'N': {'frequency': 6, 'value': 1}, 
+    'O': {'frequency': 8, 'value': 1}, 
+    'P': {'frequency': 2, 'value': 3}, 
+    'Q': {'frequency': 1, 'value': 10}, 
+    'R': {'frequency': 6, 'value': 1}, 
+    'S': {'frequency': 4, 'value': 1}, 
+    'T': {'frequency': 6, 'value': 1}, 
+    'U': {'frequency': 4, 'value': 1}, 
+    'V': {'frequency': 2, 'value': 4}, 
+    'W': {'frequency': 2, 'value': 4}, 
+    'X': {'frequency': 1, 'value': 8}, 
+    'Y': {'frequency': 2, 'value': 4}, 
+    'Z': {'frequency': 1, 'value': 10}
+}
+
+TOTAL_TILES_PER_HAND = 10
+
 def draw_letters():
-    pass
+    letter_bank = []
+    player_hand = []
+
+    # Generates a letter bank based on the letter frequency stored in LETTER_TILES
+    for letter in LETTER_TILES:
+        for i in range(LETTER_TILES[letter]['frequency']):
+            letter_bank.append(letter)
+
+    while len(player_hand) < TOTAL_TILES_PER_HAND:
```

Comment:
Since `letter_bank` is a separate data structure from `LETTER_TILES`, it's ok to remove letters from it as you go along. That way you don't have to keep comparing how often a letter was picked. Once a letter is used up, it won't be in `letter_bank` anymore and you're good!